src/commands/runDraftTool/helper/languages.ts

```diff
@@ -1,8 +1,63 @@
+import {Errorable, Succeeded} from '../../../utils/errorable';
+import {buildInfoCommand} from './draftCommandBuilder';
+import {ensureDraftBinary, runDraftCommand} from './runDraftHelper';
+
+/**
+ * The respresentation of a Draft language for use in this extension.
+ */
 export interface DraftLanguage {
    name: string;
    id: string;
    versions: string[];
 }
+
+/**
+ * The full return of the draft info command.
+ */
+interface DraftInfo {
+   supportedLanguages: DraftInfoLanguage[];
+   supportedDeployTypes: string[];
+}
+/**
+ * The respresentation of a Draft language as returned by the draft info command.
+ */
+interface DraftInfoLanguage {
+   name: string;
+   displayName: string;
+   variableExampleValues: DraftInfoExampleValues;
+}
+interface DraftInfoExampleValues {
+   // eslint-disable-next-line @typescript-eslint/naming-convention
+   VERSION: string[]; // All caps since it comes from draft builder variable conventions
+}
+export async function getDraftLanguages(): Promise<Errorable<DraftLanguage[]>> {
+   await ensureDraftBinary();
```

Comment:
☕️💡 Thoughts: this funct now return promise of `Errorable<null>` so in case of error how are we throwing from here?